src/modules/paymentProcessor/PP_Streaming_v1.sol

```diff
@@ -101,13 +101,14 @@ contract PP_Streaming_v1 is Module_v1, IPP_Streaming_v1 {
     ///         client => paymentReceiver => arrayOfStreamIdsWithPendingPayment(uint[]).
     mapping(address => mapping(address => uint[])) private activeStreams;
 
-    /// @dev    Default start, cliff and end times for new payment orders.
-    uint private defaultStart;
-    uint private defaultCliff;
-    uint private defaultEnd;
+    /// @dev    The flags that this PaymentProcessor uses
+    bytes32 internal constant processor_flags =
```

Comment:
Any reason why this is named `processor_flags` instead of `processorFlags` or (as it's a `constant`) even `PROCESSOR_FLAGS`?